tornado/iostream.py

```diff
@@ -792,7 +797,10 @@ def _find_read_pos(self):
             # _consume().
             if self._read_buffer:
                 while True:
-                    loc = self._read_buffer[0].find(self._read_delimiter)
+                    try:
+                        loc = self._read_buffer[0].find(self._read_delimiter)
+                    except AttributeError:  # might be a memoryview
+                        loc = self._read_buffer[0].tobytes().find(self._read_delimiter)
```

Comment:
Are there alternatives to `bytes.find()` that we could use instead? It looks like both `bytes` and `memoryview` support the `in` operator (although I haven't verified that it's efficient and has the semantics we want)